<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="David Montaner" />
  <title>Functional Genomics</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="index_files/reveal.js-3.3.0/css/reveal.css"/>



<link rel="stylesheet" href="index_files/reveal.js-3.3.0/css/theme/league.css" id="theme">


  <!-- some tweaks to reveal css -->
  <style type="text/css">
    .reveal h1 { font-size: 2.0em; }
    .reveal h2 { font-size: 1.5em;  }
    .reveal h3 { font-size: 1.25em;	}
    .reveal h4 { font-size: 1em;	}

    .reveal .slides>section,
    .reveal .slides>section>section {
      padding: 0px 0px;
    }



    .reveal table {
      border-width: 1px;
      border-spacing: 2px;
      border-style: dotted;
      border-color: gray;
      border-collapse: collapse;
      font-size: 0.7em;
    }

    .reveal table th {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      font-weight: bold;
      border-style: dotted;
      border-color: gray;
    }

    .reveal table td {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      border-style: dotted;
      border-color: gray;
    }

  </style>

    <style type="text/css">code{white-space: pre;}</style>

    <link rel="stylesheet" href="z_style.css"/>
    <!-- Printing and PDF exports -->
    <script>
      var link = document.createElement( 'link' );
      link.rel = 'stylesheet';
      link.type = 'text/css';
      link.href = window.location.search.match( /print-pdf/gi ) ? 'index_files/reveal.js-3.3.0/css/print/pdf.css' : 'index_files/reveal.js-3.3.0/css/print/paper.css';
      document.getElementsByTagName( 'head' )[0].appendChild( link );
    </script>
    <!--[if lt IE 9]>
    <script src="index_files/reveal.js-3.3.0/lib/js/html5shiv.js"></script>
    <![endif]-->

</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">Functional Genomics</h1>
    <h2 class="author">David Montaner</h2>
    <h3 class="date"><a href="http://www.dmontaner.com">www.dmontaner.com</a></h3>
</section>

<section><section id="from-genes-to-phenotype" class="titleslide slide level1"><h1>From Genes to Phenotype</h1></section><section class="slide level2">

<p><img src="images/dna2phenotype.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/path1.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/path2.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/path3.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/dna2phenotype.png" alt="" /></p>
</section><section id="annotation-data-bases---gene-ontology" class="slide level2">
<h1>Annotation Data Bases - Gene Ontology</h1>
<p><img src="images/dag.gif" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/dna2phenotype.png" alt="" /></p>
</section><section id="experimental-data-microarrays-ngs" class="slide level2">
<h1>Experimental Data: Microarrays &amp; NGS</h1>
<p><img src="images/array.png" alt="" /></p>
<p>thousands or millions of genomic <em>variables</em></p>
</section><section class="slide level2">

<p><img src="images/expression.png" alt="" /></p>
</section></section>
<section><section id="gene-expression-analysis" class="titleslide slide level1"><h1>Gene Expression Analysis</h1></section><section id="clustering" class="slide level2">
<h1>Clustering</h1>
<p><img src="images/clustering.png" alt="" /></p>
</section><section id="class-prediction" class="slide level2">
<h1>Class prediction</h1>
</section><section id="hypothesis-testing" class="slide level2">
<h1>Hypothesis testing</h1>
<p><img src="images/difexp.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/dag.gif" alt="" /></p>
</section></section>
<section><section id="gene-set-analysis" class="titleslide slide level1"><h1>Gene Set Analysis</h1></section><section class="slide level2">

<p><img src="images/over_representation_methods.png" alt="" /></p>
</section><section id="shift-in-the-observation-unit" class="slide level2">
<h1>Shift in the observation unit</h1>
<ul>
<li><span class="math inline">\(H_0: gene_i\)</span> is not differentially expressed …</li>
<li><span class="math inline">\(H_0: gene\ set_j\)</span> is not <strong>enriched</strong> …</li>
</ul>
<hr>
<ol type="1">
<li>more interpretable for researchers</li>
<li>greater power (sample size )</li>
<li>smaller p-value adjustment:<br />
 20,000 genes -&gt; 2,000 GO terms</li>
</ol>
<p>Takes advantage of the mildly differentially expressed genes</p>
</section><section id="gsea" class="slide level2">
<h1>GSEA</h1>
<p><img src="images/gsea.jpg" alt="" /></p>
</section><section id="drawbacks" class="slide level2">
<h1>Drawbacks</h1>
<ul>
<li>Confidence level based on a <strong>sample</strong> <em>permutation</em> test.<br />
 Computationally intensive.</li>
<li>Completely dependent of the gene level statistic.<br />
 Not very flexible: weighting, covariates, …</li>
<li>Tackles just a single <em>genomic dimension</em></li>
</ul>
</section><section id="genomic-dimensions" class="slide level2">
<h1>Genomic dimensions</h1>
<ul>
<li>Gene expression</li>
<li>Transcript expression</li>
<li>Splicing</li>
<li>microRNA activity</li>
<li>Promoters</li>
<li>Methylation</li>
<li>SNV (genetic variation)</li>
<li>Copy Number</li>
<li>Loss of heterozygosity (LOH)</li>
</ul>
</section></section>
<section><section id="logistic-regression-model" class="titleslide slide level1"><h1>Logistic Regression Model</h1></section><section class="slide level2">

<p><span class="math display">\[ \log \frac{P(g_i \in GO)}{P (g_i \notin GO)} = \kappa + \alpha \ r_i \]</span></p>
<p> </p>
<p>We model the probability of a gene belonging to a GO term<br />
as a function of its <em>score</em><br />
in the differential expression analysis</p>
</section><section class="slide level2">

<p><img src="images/over_representation_methods.png" alt="" /></p>
</section><section class="slide level2">

<p><span class="math display">\[ \log \frac{P(g_i \in GO)}{P (g_i \notin GO)} = \kappa + \alpha \ r_i \]</span></p>
<p> </p>
\begin{align}
&amp;\alpha &gt; 0 \rightarrow \text{enrichment is } + \\
&amp;\alpha &lt; 0 \rightarrow \text{enrichment is } - \\
&amp;\alpha = 0 \rightarrow \text{no enrichment}
\end{align}
</section><section id="ranking-index" class="slide level2">
<h1>Ranking index</h1>
<p>Our proposal:</p>
<p><span class="math display">\[ r = - sign (statistic) \cdot log (pvalue) \]</span></p>
<p>makes results more comparable across different studies<br />
(good for meta analysis)</p>
<p><em>quantiles</em> transformation to a <span class="math inline">\(N(0,1)\)</span><br />
reduces the effect of outliers</p>
<p> </p>
<p>But any continuous value can be used…</p>
</section></section>
<section><section id="multi-dimensional-gene-set-analysis" class="titleslide slide level1"><h1>Multi Dimensional Gene Set Analysis</h1></section><section id="genomic-dimensions-1" class="slide level2">
<h1>Genomic dimensions</h1>
<ul>
<li>Gene expression</li>
<li>Transcript expression</li>
<li>Splicing</li>
<li>microRNA activity</li>
<li>Promoters</li>
<li>Methylation</li>
<li>SNV (genetic variation)</li>
<li>Copy Number</li>
<li>Loss of heterozygosity (LOH)</li>
</ul>
</section><section class="slide level2">

<p><img src="images/GO_xl.png" alt="" /></p>
</section><section id="multidimensional-logistic-regression" class="slide level2">
<h1>Multidimensional Logistic Regression</h1>
<p> </p>
<p><span class="math display">\[ \log \frac{P(g_i \in GO)}{P (g_i \notin GO)} = \kappa + \alpha\ x_1 + \beta\ x_2 \]</span></p>
</section><section class="slide level2">

<p><img src="images/figure1.png" alt="" /></p>
</section><section id="interaction" class="slide level2">
<h1>Interaction</h1>
<p> </p>
<p><span class="math display">\[ \log \frac{P(g_i \in GO)}{P (g_i \notin GO)} = \kappa + \alpha\ x_1 + \beta\ x_2 + \gamma\ x_1  x_2 \]</span></p>
</section><section class="slide level2">

<p><img src="images/figure2a.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/figure2b.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/GO_bimodal2.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/GO_interaccion.png" alt="" /></p>
</section></section>
<section><section id="micrornas" class="titleslide slide level1"><h1>microRNAs</h1></section><section class="slide level2">

<p><img src="images/inter.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/diagram.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/interp_gen.png" alt="" /></p>
</section><section class="slide level2">

<p><img src="images/interp_go.png" alt="" /></p>
<p>mirna expression</p>
</section><section id="microrna-to-gene-transference" class="slide level2">
<h1>microRNA to gene transference</h1>
<p><span class="math display">\[ t_i = \sum_{j \in G_i} r_{j} \]</span></p>
</section><section class="slide level2">

</section></section>
<section><section id="related-topics" class="titleslide slide level1"><h1>Related topics</h1></section><section class="slide level2">

<p><img src="images/diagram.png" alt="" /></p>
</section></section>
<section><section id="and-so-summary" class="titleslide slide level1"><h1>And so… Summary</h1></section><section id="it-was-fun" class="slide level2">
<h1>It was fun</h1>
<p>… and useful</p>
<ul>
<li>New methodology and tools are available</li>
<li>My PhD</li>
<li>Supervised one more PhD and several MSc disertations</li>
<li>Teaching and communication<br />
 <a href="http://www.dmontaner.com/#Short%20Bioinformatics%20Courses">www.dmontaner.com/#Short Bioinformatics Courses</a></li>
</ul>
</section><section id="publications-peer-review" class="slide level2">
<h1>+50 Publications <em>(peer review)</em></h1>
<ul>
<li><em>Integrated gene set analysis for microRNA studies.</em> Garcia-Garcia F, Panadero J, Dopazo J, <strong>Montaner D</strong>. <a href="http://www.ncbi.nlm.nih.gov//pubmed/27324197">Bioinformatics (Oxford, England). 2016;</a></li>
<li><em>Multidimensional gene set analysis of genomic data.</em> <strong>Montaner D</strong>, Dopazo J. <a href="http://www.ncbi.nlm.nih.gov//pubmed/20436964">PloS one. 2010; 5(4):e10348.</a></li>
<li><em>Gene set internal coherence in the context of functional profiling.</em> <strong>Montaner D</strong>, Minguez P, Al-Shahrour F, Dopazo J. <a href="http://www.ncbi.nlm.nih.gov//pubmed/19397819">BMC genomics. 2009; 10:197.</a></li>
</ul>
</section><section class="slide level2">

<ul>
<li><em>Babelomics 5.0: functional interpretation for new generations of genomic data.</em> Alonso R, et al. <a href="http://www.ncbi.nlm.nih.gov//pubmed/25897133">Nucleic acids research. 2015; 43(W1):W117-21.</a></li>
<li><em>Prophet, a web-based tool for class prediction using microarray data.</em> Medina I, <strong>Montaner D</strong>, Tárraga J, Dopazo J. <a href="http://www.ncbi.nlm.nih.gov//pubmed/17138587">Bioinformatics (Oxford, England). 2007; 23(3):390-1.</a></li>
<li><em>Next station in microarray data analysis: GEPAS.</em> <strong>Montaner D</strong>, et al. <a href="http://www.ncbi.nlm.nih.gov//pubmed/16845056">Nucleic acids research. 2006; 34(Web Server issue):W486-91.</a></li>
</ul>
<p><a href="http://www.dmontaner.com/#Papers">www.dmontaner.com/#Papers</a></p>
</section><section id="software" class="slide level2">
<h1>Software</h1>
<ul>
<li><a href="http://bioconductor.org/packages/mdgsa" class="uri">http://bioconductor.org/packages/mdgsa</a></li>
<li><a href="http://bioconductor.org/packages/SNPediaR" class="uri">http://bioconductor.org/packages/SNPediaR</a></li>
<li><a href="https://github.com/dmontaner/mirbaseID" class="uri">https://github.com/dmontaner/mirbaseID</a></li>
<li><a href="https://github.com/genometra/agilent" class="uri">https://github.com/genometra/agilent</a></li>
<li><a href="https://cran.r-project.org/web/packages/TiddlyWikiR" class="uri">https://cran.r-project.org/web/packages/TiddlyWikiR</a> <a href="http://dmontaner-papers.github.io/gsa4mirna">wiki example</a></li>
</ul>
<p><a href="http://babelomics.org" class="uri">http://babelomics.org</a></p>
</section><section id="knowledge-transfer" class="slide level2">
<h1>Knowledge Transfer</h1>
<p><img src="images/logo_genometra.jpg" alt="" /></p>
<p><a href="http://www.genometra.com">www.genometra.com</a></p>
<p>First <a href="https://www.cipf.es/web/portada/spinoffs;jsessionid=C2DE665D4DFEE112095BB7B311E1A7A3">CIPF</a> Spin-off Company</p>
</section></section>
<section><section id="what-can-i-add-to-you" class="titleslide slide level1"><h1>What can I add to you</h1></section><section class="slide level2">

<ul>
<li>Statistics &amp; Machine Learning</li>
<li>Theory + applyed skills + statistical intuition</li>
<li>Used to solve <em>whole</em> problems: beyond the analysis side</li>
<li><p>Team woking</p></li>
<li><p>Very simmilar context Risk prediction in clinic an fraud …</p></li>
</ul>
</section><section class="slide level2">

<ul>
<li>Training and teaching</li>
<li>Convincing and explaining</li>
<li>Sharing</li>
</ul>
</section></section>
    </div>
  </div>

  <script src="index_files/reveal.js-3.3.0/lib/js/head.min.js"></script>
  <script src="index_files/reveal.js-3.3.0/js/reveal.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        // Push each slide change to the browser history
        history: true,
        // Vertical centering of slides
        center: true,
        // Transition style
        transition: 'convex', // none/fade/slide/convex/concave/zoom
        // Transition style for full page slide backgrounds
        backgroundTransition: 'default', // none/fade/slide/convex/concave/zoom


        // Optional reveal.js plugins
        dependencies: [
        ]
      });
    </script>
  <!-- dynamically load mathjax for compatibility with self-contained -->
  <script>
    (function () {
      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "index_files/mathjax-local/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
      document.getElementsByTagName("head")[0].appendChild(script);
    })();
  </script>

<script>
  (function() {
    if (window.jQuery) {
      Reveal.addEventListener( 'slidechanged', function(event) {  
        window.jQuery(event.previousSlide).trigger('hidden');
        window.jQuery(event.currentSlide).trigger('shown');
      });
    }
  })();
</script>


  </body>
</html>
