% Functional Genomics
% David Montaner
% [www.dmontaner.com](http://www.dmontaner.com)

From Genes to Phenotypes
==============================================================

--------------------------------------------------------------

![](images/dna2phenotype.png)

--------------------------------------------------------------

![](images/path1.png)

--------------------------------------------------------------

![](images/path2.png)

--------------------------------------------------------------

![](images/path3.png)

--------------------------------------------------------------

![](images/dna2phenotype.png)

Annotation Data Bases - Gene Ontology
--------------------------------------------------------------

![](images/dag.gif)

--------------------------------------------------------------

![](images/dna2phenotype.png)


Experimental Data: Microarrays & NGS
--------------------------------------------------------------

![](images/array.png)

thousands or millions of genomc _variables_

--------------------------------------------------------------

![](images/expression.png)


Gene Expression Analysis
==============================================================

Clustering
--------------------------------------------------------------

![](images/clustering.png)


Class prediction
--------------------------------------------------------------


Hypothesis testing
--------------------------------------------------------------

![](images/difexp.png)

--------------------------------------------------------------

![](images/dag.gif)



Gene Set Analysys
==============================================================

--------------------------------------------------------------

![](images/over_representation_methods.png)


Shift in the observation unit
--------------------------------------------------------------

- $H_0: gene_i$ is not differentially expressed ...
- $H_0: gene\ set_j$ is not __enriched__ ... 

<hr>

1. more interpretable for researchers
1. greater power (sample size )
1. smaller pvalue adjustment:  
   20,000 genes -> 2,000 GO terms

Takes advantage of the mildly differentially expressed genes

GSEA
--------------------------------------------------------------

![](images/gsea.jpg)


Drawbacks
--------------------------------------------------------------

- Confidence level based on a __sample__ _permutation_ test.  
  Computationally intensive.
- Completely dependent of the gene level statistic.  
  Not very flexible: weighting, covariates, ...
- Tackles just a single _genomic dimension_


Genomic dimensions
--------------------------------------------------------------

- Gene expresion
- Transcript expresion
- Splizing
- microRNA activity
- Promoters
- Methilation
- SNV (genetic variation)
- Copy Number
- Loss of heterozygosity (LOH)


Remarks
--------------------------------------------------------------



Logistic Regression model
==============================================================
 

Ranking index
--------------------------------------------------------------

For each gene:

$$ r = - sign (statistic) \cdot log (pvalue) $$


GSA
--------------------------------------------------------------


$$ \log \frac{P(g_i \in F)}{P (g_i \notin F)} = \kappa + \alpha \ r_i $$

* $alpha > 0$ -> association with + index
* $alpha < 0$ -> association with - index


\begin{align*}
&\alpha > 0 \rightarrow \text{asociation with } + \\
&\alpha < 0 \rightarrow \text{asociación con índice } - \\
&\alpha = 0 \rightarrow \text{no asociación}
\end{align*}
  


Multi Dimensional Gene Set Analysis
==============================================================

--------------------------------------------------------------

$$ \log \frac{\pi}{1-\pi} = \kappa + \alpha\ x_1 + \beta\ x_2 $$
    
La interpretación se realiza según el signo de los coeficientes 
$\alpha$, $\beta$ y $\gamma$.



Interaction 
--------------------------------------------------------------

$$ \log \frac{\pi}{1-\pi} = \kappa + \alpha\ x_1 + \beta\ x_2 + \gamma\ x_1  x_2 $$


microRNAs
==============================================================


microRNA to gene transference
--------------------------------------------------------------

$$ t_i = \sum_{j \in G_i} r_{j} $$












Equations
--------------------------------------------------------------


--------------------------------------------------------------

![](images/diagram.png)



Mas eq
--------------------------------------------------------------

\begin{equation}
\begin{split}
F = \{F_{x} \in  F_{c} &: (|S| > |C|) \\
 &\quad \cap (\text{minPixels}  < |S| < \text{maxPixels}) \\
 &\quad \cap (|S_{\text{conected}}| > |S| - \epsilon) \}
\end{split}
\end{equation}






And so... Summary
==============================================================

It was fun
--------------------------------------------------------------

... and useful
 
- New methodology and tools are available 
- My PhD
- Supervised one more PhD and several MSc disertations
- Teaching and communication  
  [www.dmontaner.com/#Short Bioinformatics Courses](http://www.dmontaner.com/#Short%20Bioinformatics%20Courses)


+50 Publications _(peer review)_
--------------------------------------------------------------

- _Integrated gene set analysis for microRNA studies._ Garcia-Garcia F, Panadero J, Dopazo J, **Montaner D**. [Bioinformatics (Oxford, England). 2016; ](http://www.ncbi.nlm.nih.gov//pubmed/27324197)
- _Multidimensional gene set analysis of genomic data._ **Montaner D**, Dopazo J. [PloS one. 2010; 5(4):e10348.](http://www.ncbi.nlm.nih.gov//pubmed/20436964)
- _Gene set internal coherence in the context of functional profiling._ **Montaner D**, Minguez P, Al-Shahrour F, Dopazo J. [BMC genomics. 2009; 10:197.](http://www.ncbi.nlm.nih.gov//pubmed/19397819)

--------------------------------------------------------------

- _Babelomics 5.0: functional interpretation for new generations of genomic data._ Alonso R, et al. [Nucleic acids research. 2015; 43(W1):W117-21.](http://www.ncbi.nlm.nih.gov//pubmed/25897133)
- _Prophet, a web-based tool for class prediction using microarray data._ Medina I, **Montaner D**, Tárraga J, Dopazo J. [Bioinformatics (Oxford, England). 2007; 23(3):390-1.](http://www.ncbi.nlm.nih.gov//pubmed/17138587)
- _Next station in microarray data analysis: GEPAS._ **Montaner D**, et al. [Nucleic acids research. 2006; 34(Web Server issue):W486-91.](http://www.ncbi.nlm.nih.gov//pubmed/16845056)

[www.dmontaner.com/#Papers](http://www.dmontaner.com/#Papers)


Software
--------------------------------------------------------------

- http://bioconductor.org/packages/mdgsa
- http://bioconductor.org/packages/SNPediaR
- https://github.com/dmontaner/mirbaseID
- https://github.com/genometra/agilent
- https://cran.r-project.org/web/packages/TiddlyWikiR - [ex](http://dmontaner-papers.github.io/gsa4mirna)
 
http://babelomics.org


Knowledge Transfer
--------------------------------------------------------------

![](images/logo_genometra.jpg)

[www.genometra.com](http://www.genometra.com)

First [CIPF](https://www.cipf.es/web/portada/spinoffs;jsessionid=C2DE665D4DFEE112095BB7B311E1A7A3) Spin-off Company



What can I add to you
==============================================================
--------------------------------------------------------------

- Statistics & Machine Learning
- Theory + applyed skills + statistical intuition
- Used to solve _whole_ problems: beyond the analysis side
- Team woking

- Very simmilar context Risk prediction in clinic an fraud ...

----

- Training and teaching 
- Convincing and explaining 
- Sharing

